src/App.jsx

```diff
@@ -24,22 +25,9 @@
 };
 
 export const App = () => (
-  <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">Natasha is my wife</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Olya</h2>
-      <p className="Person__partner">Maksym is my husband</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
-    </section>
+  <div>
+    <Person person={misha} />
+    <Person person={olya} />
+    <Person person={alex} />
```

Comment:
Verify that the `Person` component can handle the properties being passed to it. For instance, the `alex` object does not have a `partnerName` property, which might be required by the `Person` component.